openapi-raw.yaml

```diff
@@ -10377,7 +10387,7 @@ components:
     TemplateCreateEmbeddedResponse:
       properties:
         template:
-          $ref: '#/components/schemas/TemplateResponse'
+          $ref: '#/components/schemas/TemplateCreateEmbeddedResponse'
```

Comment:
Renaming this breaks backward compatibility

Reply:
Not yet released, merged a few hours ago: https://github.com/hellosign/hellosign-openapi/pull/306